docs/hub/model-release-checklist.md

```diff
@@ -1,102 +1,133 @@
 # Model(s) Release Checklist
 
-The Hugging Face Hub is the go-to platform for sharing machine learning models. A well-executed release can boost your model's visibility and impact. This section covers essential steps for a concise, informative, and user-friendly model release.
+The [Hugging Face Hub](https://huggingface.co/models) is the go-to platform for sharing machine learning models.
+A well-executed release can boost your model's visibility and impact. This section covers **essential** steps for a concise, informative, and user-friendly model release.
 
 ## ⏳ Preparing Your Model for Release
 
-### Uploading weights
+### Upload Model Weights
 
-When uploading models to the hub, it's recommended to follow a set of best practices:
+When uploading models to the Hub, follow these best practices:
 
-### Uploading Weights
+- **Use separate repositories for different model weights**:
+   Create individual repositories for each variant of the same architecture. This lets you group them into a [collection](https://huggingface.co/docs/hub/en/collections), which users prefer when browsing many options. It also improves visibility because each model has its own URL (`hf.co/org/model-name`) and is easier to search.
```

Comment:
```suggestion
   Create individual repositories for each variant of the same architecture. This lets you group them into a [collection](https://huggingface.co/docs/hub/en/collections), which are easier to navigate than directory listings. It also improves visibility because each model has its own URL (`hf.co/org/model-name`), makes search easier, and provides download counts for each one of your models.
```

Reply:
not sure if needed but we could also mention that it makes it easier to integrate with downstream libraries too.

Reply:
@Vaibhavs10 how do you mean? I thought downstream libraries only benefit from `transformers` integration. If you could guide me into understanding this point better I will add it here.